src/ConfigurationContext.tsx

```diff
@@ -61,6 +64,7 @@ const ConfigurationProvider: FunctionComponent = ({ children }) => {
         displayReportAnIssue,
         displaySelfAssessment,
         healthAuthorityAdviceUrl,
+        healthAuthorityLearnMoreUrl: learnMoreUrl || healthAuthorityAdviceUrl,
```

Comment:
Not sure if we should default to the value from the advice URL, maybe this learn more link should not be rendered if there's not a `LEARN_MORE_URL` I'm not sure we got clarity on this when we discussed it, might be worth circling back.